src/options.toml

```diff
@@ -34,6 +34,43 @@
 #All of these options are required
 #If they are not provided the program will exit early
 [BCs]
+    # Tells the program that we'll have changing BCs
+    # If left out of the file this defaults to a value of false
+    changing_ess_bcs = false
+    # The below gives an example of how to make use of changing essential BCs
+    # where we have the sample cyclically loaded.
+    # It would go in tandem with using the auto time stepper with a dt = 0.1
+    # and t_final = 7.0.
+    # The step number that a new BC is to be applied on.
+    # You must always have step 1 listed here, since that is the initial
+    # step that any BC is applied on.
+    # update_steps = [1, 11, 31, 51, 71]
+    # You'll want to have all the BC ids that you plan on using through out the
+    # simulation listed here. Also, when constraining the problem you'll want to
+    # take into consideration that currently we can't constrain a boundary to have
+    # a desired normal through out the deformation history.
+    #essential BC ids for the whole boundary
+    #essential_ids = [[1, 2, 3, 4],
+    #                 [1, 2, 3, 4],
+    #                 [1, 2, 3, 4],
+    #                 [1, 2, 3, 4],
+    #                 [1, 2, 3, 4]]
```

Comment:
So you need a list for each entry in update_steps? And they have to be the same list each time? The user should really have to repeat the entries if they are required to be the same throughout? 

Or the ID listings can change length -- you can drop an essential BC at some update_steps entry?

Reply:
A user does need to list the attributes for each entry in updated_steps. The reason for this is that the user can change the applied boundary condition set to whatever they want. For example, they could go from being symmetrical BCs to essential BCs at a given step. So, we don't require them to be the same through out the simulation.